docs/design/custom-xcode-schemes.md

```diff
@@ -255,9 +255,6 @@ __Values__
 - `none`: No schemes are automatically generated.
 - `all`: A scheme is generated for every buildable target even if custom schemes
   are provided.
-- `top_level_only`: A scheme is generated for every top-level target even if
-  custom schemes are provided. A top-level target in this context is one that is
-  not depended upon by any other target in the Xcode project.
```

Comment:
So, this document is a historical design document. It doesn't reflect reality. All actual docs are under `docs/bazel`.

That said, we do plan to add `top_level_only`, we just haven't gotten around to it.

I wonder if we should keep this document around? And if we shouldn't, I should probably take the info in the high level design document and make it something more up-to date at a different location.

Reply:
https://github.com/buildbuddy-io/rules_xcodeproj/blob/main/docs/bazel.md#xcodeproj-scheme_autogeneration_mode. We should update the values here to match what you found here. I didn't realize they weren't coming though.

Reply:
Ah, gotcha. I went down the path of trying it a bit and was surprised it was in the docs and not supported. Guess I wasn't properly understanding the docs.

Reply:
Ah, finally see your comment now, yes!